ArcadiaParties.Data/Repositories/AutoMapperProfile.cs

```diff
@@ -14,12 +15,33 @@ public AutoMapperProfile()
 
             CreateMap<UserDTO, UserForCalendarDTO>();
 
+            CreateMap<AssistantEmployeeDTO, UserDTO>()
+            .ConstructUsing(user => new UserDTO
```

Comment:
Code is not formatted. Is it very hard to run code reformat each time you change something in code?

Reply:
Formatted